PennMobile/Dining/SwiftUI/Views/Venue/DiningVenueView.swift

```diff
@@ -97,5 +89,11 @@ struct CustomHeader: View {
         .background(Color(UIColor.uiBackground))
         // Default Text Case for Header is Caps Lock
         .textCase(nil)
+        .sheet(isPresented: $alertAccepted) {
+            DiningLoginNavigationView(showSheetView: self.$alertAccepted)
```

Comment:
I see that you are passing down a boolean state by having a 
`@Binding var showSheetView: Bool` in your `DiningLoginNavigationView`

It should be possible to use the `@Environment(\.presentationMode) var presentationMode` to dismiss the view instead of passing down states